crates/bevy_ui/src/render/render_pass.rs

```diff
@@ -140,7 +139,7 @@ impl SortedPhaseItem for TransparentUi {
 
     #[inline]
     fn sort(items: &mut [Self]) {
-        items.sort_by_key(SortedPhaseItem::sort_key);
```

Comment:
we could use a unstable sort here because we can ensure the sort_key is unique ,but looks like I should split it to another pr to minimize the impact of current changes.